pkg/hub/repositories.go

```diff
@@ -77,12 +76,17 @@ func (c *Client) GetRepositories(account string) ([]Repository, int, error) {
 
 //RemoveRepository removes a repository on Hub
 func (c *Client) RemoveRepository(repository string) error {
-	req, err := http.NewRequest("DELETE", c.domain+fmt.Sprintf(DeleteRepositoryURL, repository), nil)
+	repositoryUrl := fmt.Sprintf("%s%s%s/%s/", c.domain, RepositoriesURL, c.account, repository)
```

Comment:
I'd like to point out that prefixing the user account to the repository name by default won't work. You can be part of many organizations, and so you may need to specify the full repository name `<org>/<repo>`.
With this code it will result with `<user>/<org>/<repo>`.
Maybe we should parse the `repository` argument, check if an org (or user) is already specified, and if not prefix with user account ? WDYT ?

Reply:
@silvin-lubecki tks for the review. That's true, I don't have access to a <org>/paid account, and I couldn't test this use-case. I'll look for a paid account, and follow up with your comments.

Reply:
@silvin-lubecki I slept on that, and I believe the cleanest way to solve is to follow a pattern for the deletion. Allowing the user to not provide a prefix(username), and request the user to provide the organization name could be confusing.

My last commit makes it clear the user needs to always provide a prefix (username or organization name) together with the repository name. WDYT?

p.s: it was tested with user repo. and organization repo. Lint issue was also fixed, I wasn't aware of the linting 🙈